### PR TITLE
Add type annotations to adafruit_clue.py

### PR DESCRIPTION
**Pull request by julianaklulo**
Adds type annotations to library `adafruit_clue.py`, solving issue #49.

### CODE REVIEW

File: adafruit_clue.py [hunk 1]
```diff
@@ -64,12 +69,12 @@ class _ClueSimpleTextDisplay:
...
     def __init__(  # pylint: disable=too-many-arguments
         self,
-        title=None,
-        title_color=0xFFFFFF,
-        title_scale=1,
-        text_scale=1,
-        font=None,
-        colors=None,
+        title: str = None,
+        title_color: Union[int, Tuple] = 0xFFFFFF,
```
Comment (tekktrik) [inline adafruit_clue.py:73]: `Tuple` is a generic, so you can add more information if you want!

Comment (tekktrik) [reply adafruit_clue.py:73]: Here's where it looks like the more detailed information can be found:  https://github.com/adafruit/Adafruit_CircuitPython_Display_Text/blob/59c27577053005435bb5466d07befa04f8d4df2f/adafruit_display_text/label.py#L51

File: adafruit_clue.py [hunk 2]
```diff
@@ -612,11 +619,11 @@ def sea_level_pressure(self):
         return self._pressure.sea_level_pressure
 
     @sea_level_pressure.setter
-    def sea_level_pressure(self, value):
+    def sea_level_pressure(self, value: int):
```
Comment (tekktrik) [inline adafruit_clue.py:622]: This may be `float`, similar to the getter property.

File: adafruit_clue.py [hunk 3]
```diff
@@ -682,20 +689,20 @@ def pixel(self):
...
-    def _generate_sample(self, length=100):
+    def _generate_sample(self, length: int = 100):
         if self._audio_out is not None:
             return
         self._sine_wave = array.array("H", self._sine_sample(length))
         self._audio_out = audiopwmio.PWMAudioOut(board.SPEAKER)
         self._sine_wave_sample = audiocore.RawSample(self._sine_wave)
 
-    def play_tone(self, frequency, duration):
+    def play_tone(self, frequency: int, duration: int):
```
Comment (tekktrik) [inline adafruit_clue.py:705]: I think both of these can be `float`.

**Review by tekktrik — CHANGES_REQUESTED** [review]
Feedback below!

File: adafruit_clue.py [hunk 4]
```diff
@@ -64,12 +69,12 @@ class _ClueSimpleTextDisplay:
 
     def __init__(  # pylint: disable=too-many-arguments
         self,
-        title=None,
-        title_color=0xFFFFFF,
-        title_scale=1,
-        text_scale=1,
-        font=None,
-        colors=None,
+        title: str = None,
```
[the remark this answers is not in the document]
Comment (julianaklulo) [reply adafruit_clue.py]: I updated it.

File: adafruit_clue.py (hunk 3 above)
Comment (julianaklulo) [reply adafruit_clue.py]: In the docstring it says that the `frequency` is `int`, so I changed `duration` to `float` and kept `frequency` as `int`.

**Review by tekktrik — APPROVED** [review]
This looks great!  Thanks so much!

File: adafruit_clue.py (hunk 1 above)
Comment (julianaklulo) [reply adafruit_clue.py]: Good suggestion, I updated it here and in the other occurencies as well.

**Pull request merged**